src/dbnode/storage/database.go

```diff
@@ -236,10 +236,18 @@ func (d *db) updateSchemaRegistry(newNamespaces namespace.Map) error {
 		curSchema, err := schemaReg.GetLatestSchema(metadata.ID())
 		if curSchema != nil {
 			curSchemaId = curSchema.DeployId()
+			if len(curSchemaId) == 0 {
+				d.log.Warn("can not update namespace schema to with empty deploy ID", zap.Stringer("namespace", metadata.ID()),
+					zap.String("current schema id", curSchemaId))
```

Comment:
I think we don't usually put spaces in key names do we?

Reply:
why not? I have been doing that. It seem to work fine.

Reply:
it works, I think we just usually use camelCase everywhere else. I'd rather it stay consistent